README.md

```diff
@@ -129,6 +129,98 @@ The image ships with some basic fonts (`font-terminus font-inconsolata font-deja
 - Alpine Linux see https://wiki.alpinelinux.org/wiki/Fonts
 - Debian/Ubuntu see https://wiki.debian.org/Fonts
 
+#### Custom fonts directory
+
+Let's say your building a docker image for you application that requires the `ffmpeg` and `ffprobe` binaries, and  want to have a dedicated directory for storing fonts, e.g. `/app/fonts`. In order to achieve this, let's take a look at the following steps.
```

Comment:
Some nit:
"Let's say you're building a docker image for your application"
"binaries, and  want" <- double space